README.md

```diff
@@ -13,14 +13,166 @@ npm install --save zeal-redux-utils
 
 ## Usage
 
-```javascript
-import {
-  createActionTypes,
-  createReducer,
-  globalizeSelectors
-} from 'zeal-redux-utils'
+zeal-redux-utils provides a number of utility functions that are useful in [Redux](http://redux.js.org/) applications, especially those that use a [modular or domain-style structure](https://jaysoo.ca/2016/02/28/organizing-redux-application/).
```

Comment:
Why is it especially useful for modular or domain-style structures? How does this make it better?